src/Compilers/Core/Portable/Syntax/InternalSyntax/SyntaxList.WithManyChildren.cs

```diff
@@ -59,20 +58,6 @@ internal WithManyChildrenBase(ObjectReader reader)
                 this.InitializeChildren();
             }
 
-            internal override void WriteTo(ObjectWriter writer)
```

Comment:
Same question above. Can we remove `internal WithManyChildrenBase(ObjectReader reader)` now?